### PR TITLE
Rename section titles with apostrophes

### LINKED ISSUE #954: Sections titles with apostrophes do not link correctly
# Bug Report

## Problem

In the Electron's documentation section, if in the navigation bar you click on the "Customizing the Application's Window Process" or on the "Customizing the Electron's Main Process" it won't link you to the appropriate section.

### What is expected to happen?

It is expected that once you click on any section in the navbar, it should link you to the appropriate section in the documentation.

### What does actually happen?

Nothing happens. The user remains in the same section.

### Information

The problem is related to the name of the section and how we build an HTML document from `.md` files. When we build an HTML document, we take the names of the headings from the `.md` file and create an anchor tag for every 2nd layer (`##`) heading.

For instance:
```
<a class="fragment-anchor" id="customizing-the-electron's-main-process"></a>
```

We use heading's name as an `id` of the element in the navbar link as well as the document, in this case, `./docs/en/latest/guide/platforms/electron/index.html`.

Since both section names "Customizing the Application's Window Process" and "Customizing the Electron's Main Process" contain an apostrophe, we also happen to use an apostrophe in the `id`. Unfortunately, this is what causes this issue.

### Possible solutions

- First and the simplest - change the name of the section.
- Manually edit the `id` of the corresponding element it in the `./docs/en/latest/guide/platforms/electron/index.html` and `href` of `<a>` element in the navbar
- Use the backslash (`\`) to escape the apostrophe symbol in the `href` of `<a>` element in the navbar

### Environment, Platform, Device

I believe this problem is across all the browsers and operating systems since they all build HTML in a similar way. I've successfully reproduced the problem on the newest Chrome and Safari browsers on macOS.

## Checklist

- [x] I searched for existing GitHub issues
- [x] I updated all Cordova tooling to the most recent version
- [x] I included all the necessary information above

### PR DESCRIPTION
**Pull request by GedasGa**
### Platforms affected

electron

### Motivation and Context

Resolves #954 

### Description

Rename section titles with apostrophes. 

### Testing

npm t

### Checklist

- [x] I've run the tests to see all new and existing tests pass
- [ ] I added automated test coverage as appropriate for this change
- [ ] Commit is prefixed with `(platform)` if this change only applies to one platform (e.g. `(android)`)
- [x] If this Pull Request resolves an issue, I linked to the issue in the text above (and used the correct [keyword to close issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/))
- [x] I've updated the documentation if necessary

### CODE REVIEW

File: www/docs/en/9.x/guide/platforms/electron/index.md [hunk 1]
```diff
@@ -144,7 +144,7 @@ By default, the `width` is set to **800** and the `height` set to **600**. This
 #### How to make my window fullscreen?
 
 Using the `fullscreen` flag property, you can force the application to launch in fullscreen.
-
+1
```
Comment (janpio) [inline www/docs/en/9.x/guide/platforms/electron/index.md]: ???

File: www/docs/en/dev/guide/platforms/electron/index.md [hunk 2]
```diff
@@ -144,7 +144,7 @@ By default, the `width` is set to **800** and the `height` set to **600**. This
 #### How to make my window fullscreen?
 
 Using the `fullscreen` flag property, you can force the application to launch in fullscreen.
-
+1
```
Comment (janpio) [inline www/docs/en/dev/guide/platforms/electron/index.md]: ???

Comment (erisu) [reply www/docs/en/dev/guide/platforms/electron/index.md]: That is weird. I scrolled though the commit and didn't see this haha.

Comment (GedasGa) [reply www/docs/en/dev/guide/platforms/electron/index.md]: I don't know how this slipped through. I can make a cleanup PR.

**Pull request merged**